### PR TITLE
docs: AP-5783 add QuickSight user docs

### PR DESCRIPTION
**Pull request by tom-webber**
This PR relates to ministryofjustice/analytical-platform#5783

### CODE REVIEW

File: source/documentation/tools/quicksight/02-overview.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+We’re still in beta so we'd love to get some of you using QuickSight to get your feedback to help guide data visualization best practices in the Ministry of Justice and make sure we can continue to improve the user experience.
+
+## What data is available in QuickSight?
+QuickSight is connected to [the `mojap-derived-table` domains] published through [Create a Derived Table](/tools/create-a-derived-table). 
+
+If the output from Create a Derived Table goes to the `mojap_derived_tables/prod/models` path in S3, it should be available in QuickSight.
+
```
Comment (sdurnberger) [inline source/documentation/tools/quicksight/02-overview.md:12]: I think you can probably remove this section and combine it with your overview at the top of the page! It's like there are 2 overviews at the moment. You could cut this down a bit to focus on what users can do supported by your AWS links where they can learn more. 

Suggest adding the following to the overview to replace the data question:
If you're using QuickSight on the Analytical Platform, you can only use data published through [Create a Derived Table](/tools/create-a-derived-table).

File: source/documentation/tools/quicksight/01-index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# QuickSight
```
Comment (sdurnberger) [inline source/documentation/tools/quicksight/01-index.md:1]: Could this be more descriptive? For example: 'Use QuickSight to visualise your data' or similar. That could give users better understanding of whether this content is relevant to them.

File: source/documentation/tools/quicksight/01-index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+⚠️ This service is in beta ⚠️
+
+QuickSight is a data visualisation service that allows you to create interactive dashboards and reports from data available to the Analytical Platform.
+
+We’re still in beta so we'd love to get some of you using QuickSight to get your feedback to help guide data visualization best practices in the Ministry of Justice and make sure we can continue to improve the user experience.
```
Comment (sdurnberger) [inline source/documentation/tools/quicksight/01-index.md:7]: Is there a feedback mechanism (or placeholder if it's to be decided) to put here so it's clear how people can provide it?

File: source/documentation/tools/quicksight/04-working-with-quicksight.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Working with QuickSight
+
+## Finding datasets
```
Comment (sdurnberger) [inline source/documentation/tools/quicksight/04-working-with-quicksight.md:3]: It looks like 'Add a dataset' or 'Add datasets' might be more accurate for what this section covers :)

File: source/documentation/tools/quicksight/04-working-with-quicksight.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Adding a dataset in this way enables you to add a single table at a time.
+
+You'll be asked if you want to 'Import to [SPICE] for quicker analytics'.  
+This imports the data into memory, rather than querying the data where it lives in S3. Whether to use SPICE or directly query the data is a judgement call, and will likely come down to:
+
+- how complex the analysis is
+- the size, format and structure of the underlying data
+- how likely users are to drill down into the dashboards
+- user tolerance for load times while making changes to dashboards
```
Comment (sdurnberger) [inline source/documentation/tools/quicksight/04-working-with-quicksight.md:28]: Is it possible to flip this approach? For example, giving users a recommendation for the factors you've described. 'You should not use SPICE if...'

File: source/documentation/tools/quicksight/01-index.md (hunk 3 above)
Comment (tom-webber) [reply source/documentation/tools/quicksight/01-index.md:7]: James has just added it, so yep! Linked.

File: source/documentation/tools/quicksight/04-working-with-quicksight.md (hunk 5 above)
Comment (tom-webber) [reply source/documentation/tools/quicksight/04-working-with-quicksight.md:28]: I like this idea. My suggestion would basically boil down to 'SPICE costs money, don't use it if you don't need it'. Due to lack of familiarity with the impacts it makes on user experience, I don't feel knowledgable enough for that approach, and would prefer to leave it in the users' hands.

**Review by julialawrence — APPROVED** [review]
LGTM

**Pull request merged**